src/content/docs/errors-inbox/legacyerrors-eol.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Getting started with errors inbox'
+metaDescription: "Errors inbox is an error tracking solution designed to give you the tools to proactively detect, triage, and resolve errors across your full application stack."
+freshnessValidatedDate: never
+---
+
+We're excited to introduce a new and improved errors experience, designed to provide a more comprehensive and intuitive way to manage errors in your applications. To ensure a smooth transition, please take note of the following important updates:
+
+## Important Dates:
+
+* **January 31, 2024:** The legacy errors experience will be discontinued.
```

Comment:
I'm not sure what communications exist around this. EOL comms may or may not have covered this, but defining what the legacy experience is and what the 'new' experience is in 2-3 sentences up to a paragraph might be helpful. How do I know if I'm on the legacy experience? What does legacy mean, etc

Reply:
@meda123 has this been covered in broader comms?